py/src/braintrust/framework.py

```diff
@@ -91,8 +91,8 @@ class EvalScorerArgs(SerializableDataClass):
 
 EvalScorer = Union[
     Scorer,
-    Callable[[Input, Output, Output], Score],
-    Callable[[Input, Output, Output], Awaitable[Score]],
+    Callable[[Input, Output, Output], Optional[Score]],
```

Comment:
Should we have symmetrical logic here for allowing raw number outputs for the scorers?